biggerquery/configuration.py

```diff
@@ -6,8 +6,16 @@
 from .utils import unzip_file_and_save_outside_zip_as_tmp_file
 
 
-class DatasetConfig(object):
+class BiggerQueryConfig(object):
```

Comment:
Maybe just Config? Similarly to spring @Configuration (not SpringConfiguration).